rawpheno.install

```diff
@@ -386,18 +395,26 @@ function rawpheno_install() {
   $arr_cvtermid_unit = array();
 
   foreach($arr_unit as $unit => $def) {
-    $tmp = tripal_insert_cvterm(
-      array(
-        'id' => 'rawpheno_tripal:' . $unit,
-        'name' => $unit,
-        'definition' => $def,
-        'cv_name' => $cv['cv_unit']
-      )
+    $param = array(
+      'id' => 'rawpheno_tripal:' . $unit,
+      'name' => $unit,
+      'definition' => $def,
+      'cv_name' => $cv['cv_unit']
     );
 
-    $arr_cvtermid_unit[$unit] = $tmp->cvterm_id;
+    if (function_exists('chado_insert_cvterm')) {
+      $tmp = chado_insert_cvterm($param);
+    }
+    else {
+      $tmp = tripal_insert_cvterm($param);
+    }
+
+    if ($tmp) {
+      $arr_cvtermid_unit[$unit] = $tmp->cvterm_id;
+    }
```

Comment:
Everytime you use an if like this, you should have an else with a tripal_report_error(). Otherwise, you may be masking the problem.